contents/tutorials/supabase-query.md

```diff
@@ -0,0 +1,166 @@
+---
+title: How to query Supabase in PostHog
+date: 2024-08-01
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+Combining your app and analytics data is a powerful way to understand your product. Supabase is a popular choice for handling that app data, as it provides a database, auth, storage, and more all-in-one.
+
+Because Supabase is built on Postgres, we can link and query it in PostHog using our data warehouse. This tutorial shows you how to do that and provides some example insights you can create with the data afterward. 
+
+## Linking Supabase data to PostHog
+
+To start, you need both a Supabase and PostHog account. Once you have those, head to PostHog's [data warehouse tab](https://us.posthog.com/data-warehouse) and:
+
+1. Click **Link source.**
+2. Choose the **Postgres** option by clicking **Link.**
+3. Go to your Supabase project settings and under configuration, click **Database.**
+4. Fill out the PostHog data warehouse link fields with your Supabase data.
```

Comment:
This was confusing for me. I think we should first tell people to go to supabase and copy this data, and then go to posthog. It should read better this way, and involve less switching between apps

Reply:
This would require going to Supabase first copying a selection of data on the database page to somewhere and then copying those individual pieces of data to PostHog. 

It requires a lot of manual copying and pasting but I think this is the best way.